simtbx/diffBragg/src/diffuse_util.h

```diff
@@ -0,0 +1,639 @@
+#ifndef SIMTBX_DIFFBRAGG_DIFFUSE_UTIL
+#define SIMTBX_DIFFBRAGG_DIFFUSE_UTIL
+
+#include <simtbx/diffBragg/src/diffuse_util.h>
```

Comment:
The file currently includes itself, did you mean to include utils.h?

Reply:
This has been addressed. thanks